src/acl.c

```diff
@@ -210,7 +210,7 @@ int ACLSetSelector(aclSelector *selector, const char *op, size_t oplen);
  * information about the nature of the strings just monitoring the execution
  * time of the function. Note: The two strings must be the same length.
  */
-int time_independent_strcmp(char *a, char *b, int len) {
+int time_independent_strcmp(const char *a, const char *b, int len) {
```

Comment:
> set read-only pointer parameter to const.

There are a lot of such usages in the code base, i don't think we want to do this minor cleanup (i break the git blame log and it is not necessary)

Reply:
I have reverted the changes, thanks for your review.